docs/Limitations.md

```diff
@@ -151,35 +120,6 @@ moment.
 See [this issue](https://github.com/kata-containers/runtime/issues/2812) for more details.
 [Another issue](https://github.com/kata-containers/kata-containers/issues/1728) focuses on the case of `emptyDir`.
 
-
-## Host resource sharing
-
-### docker run --privileged
-
-Privileged support in Kata is essentially different from `runc` containers.
-Kata does support `docker run --privileged` command, but in this case full access
-to the guest VM is provided in addition to some host access.
-
-The container runs with elevated capabilities within the guest and is granted
-access to guest devices instead of the host devices.
-This is also true with using `securityContext privileged=true` with Kubernetes.
-
-The container may also be granted full access to a subset of host devices
-(https://github.com/kata-containers/runtime/issues/1568).
-
-See [Privileged Kata Containers](how-to/privileged.md) for how to configure some of this behavior.
```

Comment:
This one I'd also move to the current limitations, removing the `docker run --privileged` part, but mentioning that this is a limitation with privileged containers.

Reply:
@fidencio changes applied